docs/src/docs/asciidoc/index.adoc

```diff
@@ -19,6 +19,12 @@ If you are using alternative build systems, see <<alternative-build-systems.adoc
 [[changelog]]
 == Changelog
 
+=== Release 0.10.2
```

Comment:
Actually, we already released `0.10.2` version (but apparently we didn't add proper changelog entries in any of the PRs merged for that release)  so your changes belong to `0.10.3` version. Regarding missing `0.10.2` entries, I can create a PR to add them

Reply:
Sorry I had fixed that already but pushed on the wrong remote :facepalm: